### ISSUE TITLE
vpk non-interactive mode 

### ISSUE THREAD

**Issue by AArnott** [velopack/velopack#78]
While I wouldn't expect a CI to accidentally pack twice, it's disturbing that the tool is interactive without a switch to non-interactive mode. This is what I see:

```
QUESTION:
A release in this channel with the same or greater version already exists. Do you want to continue and potentially
overwrite files? [y/n] (n):
```

Can we add a `--force` switch or something to avoid the questions and assume affirmative answers?

**Comment by caesay** [comment]
Hi, thanks for trying out Velopack. There is a non interactive flag (`-x`), in which case the prompt answers default to No. You can add the (`-y`) flag to answer Yes. These can be combined: `vpk pack -xy ...`.

**Comment by AArnott** [comment]
Thanks. I see now that you're right. The usage docs _do_ reveal those switches, but only if you don't include the `pack` command before `-H`. That's why I missed it the first time.

I never saw the interactive prompt in CI. I'm only now preparing my CI to run this. 

Awesome package. It has so many missing features from squirrel that I had to fill in myself or make do without. Great work!

**Comment by caesay** [comment]
Thanks for the kind words. Yes, having the root options appear globally is on my list to fix! Out of curiosity, what features does Velopack have that excite you the most / you didn't have before?

**Comment by AArnott** [comment]
IIRC built-in channel support wasn't in squirrel. I think I implemented it myself just by diverting to sub-paths in the URL.
I really like how your installer can chain in the .NET runtime installer. Without this, in Squirrel I had to use self-contained apps, which severely bloated my payload.
But the irreplaceable, cost saving feature is your `--keepMaxReleases` switch, since I really don't care to have more than several releases saved in the cloud. Anyone who doesn't upgrade frequently enough can just download the full latest payload rather than benefit from the deltas, rather than me pay an ever-increasing amount for storage.

BTW on the channel support, I searched the docs but ultimately had to resort to experimentation to determine whether it was safe to mix multiple channels into the same directory.
